doc/sphinx-guides/source/installation/config.rst

```diff
@@ -976,9 +982,9 @@ The Google Cloud Archiver can send Dataverse Project Bags to a bucket in Google'
 
 ``curl http://localhost:8080/api/admin/settings/:ArchiverClassName -X PUT -d "edu.harvard.iq.dataverse.engine.command.impl.GoogleCloudSubmitToArchiveCommand"``
 
-``curl http://localhost:8080/api/admin/settings/:ArchiverSettings -X PUT -d ":GoogleCloudBucket, :GoogleCloudProject"``
+``curl http://localhost:8080/api/admin/settings/:ArchiverSettings -X PUT -d ":GoogleCloudBucket, :GoogleCloudProject, :BagGeneratorThreads"``
 
-The Google Cloud Archiver defines two custom settings, both are required. The credentials for your account, in the form of a json key file, must also be obtained and stored locally (see below):
+The Google Cloud Archiver defines two custom settings, both are required. It can also use the :BagGeneratorThreads setting as described in the DuraCloud Archiver section above. The credentials for your account, in the form of a json key file, must also be obtained and stored locally (see below):
```

Comment:
Here and above is a reference to the DuraCloud archiver section but technically the section is called "DuraCloud Configuration". Not a big deal. I'm sure people will find it.

Can we also document `:BagGeneratorThreads` under the exhaustive list of database settings at the bottom? Other related BagIt settings are also documented there. We can also create a custom anchor and use `:ref:` to link to it to make it easier to link to, if you want.

Also, our release notes point out new database settings. Can you please create a release note file about the new setting (and anything else you might like to say)? (I saw your comment that multiple BagIt changes are coming but that's fine. Maybe we can consolidate release notes as we go along.)

Reply:
Thanks @pdurbin  - I've made the doc changes as requested and also updated the overall integrations section to talk about Bagit/BagPack export since we now have 3 archivers (and 5 soon!) rather than just DuraCloud.

Reply:
Wow, five. Nice.

Reply:
Oh, and thanks for creating an issue about release notes for all these changes:

- #8611